### PR TITLE
Port unit.js to @flow.

### LINKED ISSUE #854: Port `units.js` to @flow
Technically, this soft-requires #848 since `units.js` uses the `Option` type. However, it seems to work even without that.

### PR DESCRIPTION
**Pull request by marcianx**
Fixes #854.

### CODE REVIEW

File: src/units.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 
 import ParseError from "./ParseError";
+import Options from "./Options";
 
 // This table gives the number of TeX pts in one of each *absolute* TeX unit.
 // Thus, multiplying a length by this number converts the length from units
 // into pts.  Dividing the result by ptPerEm gives the number of ems
 // *assuming* a font size of ptPerEm (normal size, normal style).
-const ptPerUnit = {
+const ptPerUnit: {[string]: number} = {
```
Comment (kevinbarabash) [inline src/units.js]: Is this necessary?

Comment (xsznix) [reply src/units.js]: No; Flow will infer the type automatically. Ditto for line 34 as well.

Comment (xsznix) [reply src/units.js]: hm, on second thought, Flow isn't as good at inferring types when you use computed properties, so I think it's probably good to keep this annotation and the one below.

Comment (marcianx) [reply src/units.js]: Removed this and the one on line 34.

**Pull request merged**